Changelog.md

```diff
@@ -1,4 +1,13 @@
 # Next
+- Move project to using a single target for all platforms
+
+# 8.0.3
```

Comment:
The entries in the changelog are incorrect. I think we only need the one under `Next`.